JitsiConference.js

```diff
@@ -591,6 +620,62 @@ JitsiConference.prototype._setupNewTrack = function (newTrack) {
 };
 
 /**
+ * Adds loca WebRTC stream to the conference.
+ * @param {MediaStream} stream new stream that will be added.
+ * @param {function} callback callback executed after successful stream addition.
+ * @param {function(error)} errorCallback callback executed if stream addition fail.
+ * @param {object} ssrcInfo object with information about the SSRCs associated with the
+ * stream.
+ * @param {boolean} [dontModifySources] if true _modifySources won't be called.
```

Comment:
In fact this whole method is kind of "lib-jitsi-meet" internal. I will rename it this way. I will describe when it can be true.